src/main/kotlin/uk/gov/justice/hmpps/offenderevents/service/OffenderEventStore.kt

```diff
@@ -13,8 +15,8 @@ import uk.gov.justice.hmpps.offenderevents.service.OffenderEventStore.Topics.UNK
 @Service
 class OffenderEventStore(
   @Value("\${model.cacheSize}") private val cacheSize: Int,
-  private val store: MutableList<DisplayMessage> = mutableListOf()
-) : MutableList<DisplayMessage> by store {
```

Comment:
delegation here meant that we couldn't use mockito to mock out the class as some of the delegated methods were final

Reply:
however the only change to the code by doing so was actually to remove an `override`, main changes were then in the tests